services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/HankeServiceITests.kt

```diff
@@ -113,6 +119,50 @@ class HankeServiceITests : DatabaseTest() {
         confirmVerified(cableReportService)
     }
 
+    @Nested
+    inner class GetHankeId {
+        private val hankeTunnus = "HAI23-41"
+
+        @Test
+        fun `Returns null if hanke not found`() {
+            val response = hankeService.getHankeId(hankeTunnus)
+
+            assertThat(response).isNull()
+        }
+
+        @Test
+        fun `Returns hanke id if hanke found`() {
+            val hanke = hankeFactory.save()
+
+            val response = hankeService.getHankeId(hanke.hankeTunnus!!)
+
+            assertThat(response).isEqualTo(hanke.id!!)
+        }
+    }
+
+    @Nested
+    inner class GetHankeIdOrThrow {
+        private val hankeTunnus = "HAI23-41"
```

Comment:
These could possibly be declared once outside of inner classes.

Reply:
Moved to top level